src/Microsoft.AspNetCore.Blazor/Json/SimpleJson/SimpleJson.cs

```diff
@@ -1430,7 +1430,13 @@ public virtual object DeserializeObject(object value, Type type)
                             obj = value;
                         else
                         {
-                            obj = ConstructorCache[type]();
+                            try{
```

Comment:
The brace style convention for ASP.NET repo projects project is:

```
try
{
    ... some code ...
}
catch (...) // Note the space after 'catch'
{
    ... some code ...
}
```

Could you update the PR to match?